CONTRIBUTING.md

```diff
@@ -0,0 +1,86 @@
+# Contributing to Poutyne
+We love your input! We want to make contributing to this project as easy and transparent as possible, whether it's:
+
+- Reporting a bug
+- Discussing the current state of the code
+- Submitting a fix
+- Proposing new features
+- Becoming a maintainer
+
+## We Develop with Github
+We use github to host code, to track issues and feature requests, as well as accept pull requests.
+
+## We Use [Github Flow](https://guides.github.com/introduction/flow/index.html), So All Code Changes Happen Through Pull Requests
+Pull requests are the best way to propose changes to the codebase. We actively welcome your pull requests:
+
+1. Fork the repo and create your branch from `master`.
+2. If you've added code that should be tested, add tests.
+3. If you've changed APIs, update the documentation.
+4. Ensure the test suite passes.
+5. Make sure your code lints.
```

Comment:
Should not we use a linter in the .yml of travis ? (easier and more consistent with our style)

Reply:
I'm no expert with Travis. Can we automatically run the linting and/or the formatting with Travis. 

Let's say we add the linting, how does the contributor see the results ?

Reply:
We probably can run the linting in Travis but I don't know if there is a simple way to make it fail the tests if linting does not pass. Also, if we do that, the pylint version should be fixed to some version because each version tends to add warnings that were not there previously.

With yapf, is it possible to just test the formatting and not modify the files? If so, it can be a good idea to add it to the Travis script.

Anyway, I don't think that these modifications fall in the scope of this pull request.

Reply:
Agreed, the Travis integration is interesting but out of scope here.

Reply:
I'll take a look for the linter integration also, just to be sure we don't miss any bad syntax.